dev/io.openliberty.webcontainer.servlet.6.1.internal_fat/publish/servers/servlet61_AddAndSetHeaderTest/bootstrap.properties

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+bootstrap.include=../testports.properties
+osgi.console=7777
+com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:GenericBNF=all:io.openliberty.webcontainer*=all
```

Comment:
Do we need all of this tracing? Would you be comfortable cutting this down or removing it? We can always add it in if we have a build break and need further debug in a build.